src/App.tsx

```diff
@@ -1,18 +1,31 @@
-import React from 'react';
+import React, {useState} from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
+  .map((n, index) => ({ name: `Item ${n}`, id: index }));
 
 export const App: React.FC = () => {
+  const numbersOfPages = 9;
+  const [itemsPerPages, setItemsPerPages] = useState(5);
```

Comment:
```suggestion
  const [itemsPerPages, setItemsPerPages] = useState(5);
  const numbersOfPages = Math.ceil(items.length / itemsPerPages);
```
`numbersOfPages` now is a static value, which is wrong
you may still make it dynamic without a separate state for it